calc/calc.py

```diff
@@ -53,16 +50,36 @@ class Calc:
     7
 
     """
+    # Rabbit Hole:
+    #    You can read the position=0 syntax as saying "The default argument,
+    #    if you don't pass an argument to this function, is the value on the
+    #    right.
     def __init__(self, text, position=0, current_token=None):
-        """Constructor for a :class:`Calc` object."""
-        self.text = text
-        self.position = position
+        """Set's the text, position, and current_token attributes for the
+        Calc class.
+
+        Args:
+        text (str): The text to be interpreted.
+        position (int):
```

Comment:
Since these arguments are optional I find it nice to include that in the type field like `position (int, optional):`. I think the Sphinx Napoleon Google docstrings examples use that.